test/models/components_test.js

```diff
@@ -1,4 +1,5 @@
 const { expect } = require('chai');
+const { it } = require('mocha');
```

Comment:
Why did you need to import `it`? I think we can remove that.

Reply:
You're right.